gcp_variant_transforms/libs/variant_merge/merge_with_non_variants_strategy.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Variant merge stategy that can handle both Variants and Non-variants."""
+
+from __future__ import absolute_import
+
+from intervaltree import IntervalTree
+
+from gcp_variant_transforms.beam_io.vcfio import Variant
+from gcp_variant_transforms.libs.variant_merge import move_to_calls_strategy
+from gcp_variant_transforms.libs.variant_merge import variant_merge_strategy
+
+__all__ = ['MergeWithNonVariantsStrategy']
+
+DEFAULT_WINDOW_SIZE = 2500
+
+
+class MergeWithNonVariantsStrategy(variant_merge_strategy.VariantMergeStrategy):
+  """A merging strategy that can handle non-variant records.
+  Variants will be merged across files by using reference_name and start
```

Comment:
nit: please add an empty line after the first line.

Reply:
Done.